### ISSUE TITLE
Documentation is sparse. question about parameters

### ISSUE THREAD

**Issue by Firehawke** [b4rtaz/voice-assistant#6]
What would be the way to include a parameter with an action?

For instance, if I want to go to the next instance of a keyword using ``workbench.action.findInFiles `` as per https://stackoverflow.com/questions/62251045/search-find-in-files-keybinding-can-take-arguments-workbench-view-search-can, how would it be formatted in the .json, assuming it can?

**Comment by b4rtaz** [comment]
I've just published 0.1.1 version. I've added `args` parameter to `voice-assistant.json`.

```json
{
    "commands": [
        {
            "command": "Do find class",
            "action": "vsc-command:workbench.action.findInFiles",
            "args": [ { "query": "class" } ]
        }
    ]
}
```

**Comment by Firehawke** [comment]
Thanks! I'll give it a try tonight, much appreciated!
